### DISCUSSION TITLE
HTTP add-on: Error in interceptor while waiting for the backend pod to be up and running.

### DISCUSSION THREAD

**Discussion by hgpepco** [kedacore/keda#3266] [Q&A / Need Help] [answered]
Currently trying keda http-add-on v0.3. and keda v2.5.1 I am facing 2 issues,
1. No healthy upstream error for the first try and the subsequent tries, the request is served.(Updated the KEDA_CONDITION_WAIT_TIMEOUT)
2. In the interceptor I see this error when it need to wait for the backend to be up and running. Also below is the interceptor configuration.

`{"level":"error","ts":1655424772.5865734,"logger":"runProxyServer","caller":"interceptor/proxy_handlers.go:83","msg":"wait function failed, not forwarding request","error":"context marked done while waiting for deployment func-httptrigger-test to reach > 0 replicas (context canceled)","stacktrace":"main.newForwardingHandler.func1\n\t/go/src/github.com/kedacore/http-add-on/interceptor/proxy_handlers.go:83\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2046\nmain.countMiddleware.func1\n\t/go/src/github.com/kedacore/http-add-on/interceptor/middleware.go:48\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2046\nnet/http.serverHandler.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/usr/local/go/src/net/http/server.go:1929"}
![image](https://user-images.githubusercontent.com/97607131/175121923-35f2a643-c1f3-41d4-9e34-bcef22129bf7.png)
`

**Comment by tomkerkhove** [comment, marked as answer]
Resolving as this has been cross-posted 3 times.
